.gitignore

```diff
@@ -57,3 +57,9 @@ docs/source/_templates/
 
 #VSCode IDE
 .vscode/
+
+# Virtual environment
+venv*
```

Comment:
Interesting, when I use `virtualenv` it automatically adds `.gitignore` file:
```bash
akukushkin@t14s:~/git/patroni$ virtualenv venv
created virtual environment CPython3.10.6.final.0-64 in 169ms
  creator CPython3Posix(dest=/home/akukushkin/git/patroni/venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/akukushkin/.local/share/virtualenv)
    added seed packages: pip==23.1.2, setuptools==67.7.2, wheel==0.40.0
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
akukushkin@t14s:~/git/patroni$ cat venv/.gitignore
# created by virtualenv automatically
*
```

But, `python -m venv venv` doesn't :(

Anyway, for directories we need to use the following:
```suggestion
venv*/
```

Reply:
Strange! Wonder why that is.